src/mono/wasm/debugger/BrowserDebugProxy/DevToolsHelper.cs

```diff
@@ -149,6 +151,89 @@ public static Result FromJson(JObject obj)
             var result = obj["result"] as JObject;
             return new Result(result, false);
         }
+        public static Result FromJsonFirefox(JObject obj)
+        {
+            //Log ("protocol", $"from result: {obj}");
+            JObject o;
+            if (obj["ownProperties"] != null && obj["prototype"]?["class"]?.Value<string>() == "Array")
+            {
+                var ret = new JArray();
+                var arrayItems = obj["ownProperties"];
+                foreach (JProperty arrayItem in arrayItems)
+                {
+                    if (arrayItem.Name != "length")
+                        ret.Add(arrayItem.Value["value"]);
+                }
+                o = JObject.FromObject(new
+                {
+                    result = JObject.FromObject(new
+                    {
+                        value = ret
+                    })
+                });
```

Comment:
```suggestion
                o = JObject.FromObject(new
                {
                    result = new
                    {
                        value = ret
                    }
                });
```
You can nest anonymous objects rather than calling FromObject multiple times